src/app/workflow/versions/versions.component.html

```diff
@@ -210,7 +210,20 @@
         >
       </th>
       <td mat-cell *matCellDef="let version">
-        <mat-icon data-cy="metrics" *ngIf="version.metricsByPlatform && (version.metricsByPlatform | json) !== '{}'">check</mat-icon>
+        <img
+          *ngIf="showExecutionMetricsIcon(version)"
```

Comment:
Best practice is that you should use a pipe. Up to the team to decide if it's necessary or not.